"Completed k-means clustering"`

MarcFalic opened the pull request:
Applied Standardization
Possible issue = misalignment of dataframes, kmeans.fit_predict(X_scaled) is applied to df_selected, but results are stored in df["cluster"]. Resulting on the clusters label might not allign correctly.
The sample is applied on the whole dataset, and not the selected 100.

MarcFalic committed:
Add files via upload

MarcFalic commented:
@jvnthny-espiritu

jvnthny-espiritu commented:
kindly do the actual implementation boss, wag daw gagamit ng libraries ng kmeans

jvnthny-espiritu reviewed (changes requested):
Kindly do implementation. Do not use KMeans function from scikit library. [optional] do the elbow method to find the most optimal k.

MarcFalic committed:
Add files via upload

Set number of clusters to 5
randomly picked 5 points from the dataset to serve as a starting centroids
computed the euclidean distance of  every data point
re computed the centroids as the mean of all points in each cluster
using the Within-Cluster Sum of Squares, we measured how the clustering was done.
provided elbow method
